components/init/node/init.mjs

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs';
```

Comment:
I deliberately named this file `init.mjs` rather than `index.mjs`. Having the source for each component in a file named `index.mjs` makes them very hard to keep track of.

Reply:
The files in `dist` are generated by the build system with the command `npm run build`. You need to update `build/component/index.mjs` to register your new entry point. This automated file generation is the reason why all components need to have their entry file named `index.mjs`. I understand your concern about having many files named `index.mjs`. It could be possible to parametrize this in the `.build.yml` file each component. But, in my opinion, they are things more urgent to do first.

Reply:
If the files in `dist` are generated by the build, should they be removed from the repo and `/dist/` added to `.gitignore`?

Reply:
Indeed! Done in:
1. c6a434535f4b6316905fb16917af9ce6c046e5ad
2. b89d63a0d0f1dfd6d7f6e79fbe6060fbb5a5b53f
3. 2520cf7d0fef6d0f5f9b533e930fc4eca5c16050